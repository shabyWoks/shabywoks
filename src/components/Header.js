import React from 'react';
import {Link} from 'react-router-dom';

const Header = (props) => {
    return (
        <div>
            <ul>
                <Link to="/" > Dashboard </Link>
                <Link to="/my-library" > My Library </Link>
            </ul>
        </div>
    )
}

export default Header;