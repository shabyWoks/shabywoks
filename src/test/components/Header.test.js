import Header from '../../components/Header';
import {shallow} from 'enzyme';
import React from 'react';

test('should render dashboard correctly', () => {
    // const wrapper = shallow(<Header />);
    // expect(wrapper).toMatchSnapshot();
});