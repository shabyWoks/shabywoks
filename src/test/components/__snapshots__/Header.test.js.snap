// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul>
        <Link
          replace={false}
          to="/"
        >
           Dashboard 
        </Link>
        <Link
          replace={false}
          to="/my-library"
        >
           My Library 
        </Link>
      </ul>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            replace={false}
            to="/"
          >
             Dashboard 
          </Link>,
          <Link
            replace={false}
            to="/my-library"
          >
             My Library 
          </Link>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": " Dashboard ",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": " Dashboard ",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": " My Library ",
            "replace": false,
            "to": "/my-library",
          },
          "ref": null,
          "rendered": " My Library ",
          "type": [Function],
        },
      ],
      "type": "ul",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul>
          <Link
            replace={false}
            to="/"
          >
             Dashboard 
          </Link>
          <Link
            replace={false}
            to="/my-library"
          >
             My Library 
          </Link>
        </ul>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Link
              replace={false}
              to="/"
            >
               Dashboard 
            </Link>,
            <Link
              replace={false}
              to="/my-library"
            >
               My Library 
            </Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": " Dashboard ",
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": " Dashboard ",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": " My Library ",
              "replace": false,
              "to": "/my-library",
            },
            "ref": null,
            "rendered": " My Library ",
            "type": [Function],
          },
        ],
        "type": "ul",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
